### ISSUE TITLE
Normalization layer for magnet

### ISSUE THREAD

**Issue by badulion** [MAGNET4Cardiac7T/magnet-pinn#20]
Both the input features as well as the target variables need to be normalized. Whats needed:

- abstract class InputNormalizer (torch.nn.Module)
    - different types of InputNormalization(log, standard, min-max)
- abstract class FieldNormalizer (torch.nn.Module) -> forward and inverse
    - different types of FieldNormalizer (global vs per-voxel, log, standard, min-max)
